### ISSUE TITLE
"dubstep yoi yoi" preset is a bit louder than the other presets

### ISSUE THREAD

**Issue by StinkerB06** [johnnesky/beepbox#28]

**Comment by Micasddsa** [comment]
Yea just lower the volume. simple enough. This shouldn't be an issue. You should close the issue now.

**Comment by StinkerB06** [comment]
The point of this issue is that I want to know why John Nesky made the preset louder, like if it's intentional or not.

**Comment by Micasddsa** [comment]
Ok took a look at it, it's most likely not intentional and is just because of the bit crush effect applied on it.

**Comment by StinkerB06** [comment]
@Micasddsa Nah, I think it's because of the note filter. Here's a demo comparison of a random preset, "dubstep yoi yoi", and an edit that *only* has the bitcrusher effect disabled:
https://www.beepbox.co/#9n10s0k0l00e02t22a7g02j02r1i12o3T5v1u88f0q0x10r81d35HKT-DRJABBBszrrh2E1b2T0v1u59f10r6qgM10de3a0d23w5h0E1c04T0v1u00f10r6qgw10de3d23w5h0E1c04brp1AarnU9ILijh_rqq_iAnCKdUTtUSSVHzudTudJ

This is what the rendered audio looks like:
![0091](https://user-images.githubusercontent.com/30059831/172469533-85eddd75-bb6c-4355-ae9b-2ccd913644cf.png)

In conclusion, I believe this might not be an issue with certain speakers, but it definitely is for certain others.

**Comment by Micasddsa** [comment]
_man you really went audacity with that. i just used my ears. but anyways_
But yea, taking another look, It's the note filter, i do hear that, even without the note filter, the bit crush makes it slightly louder than normal instruments but not enough to really make an issue about.
I'm pretty sure I've heard before that beepbox's master compressor is sometimes wonky so it might be that.
But if we are talking about the "Dubset Yoi Yoi" preset, it's probably unintentional as it's the note filter that's causing the issue. Pretty sure though that this might be the start of another issue though about the (Maybe) compressor or the note filter. idk how beepbox really works under the hood that much just yet.

**Comment by johnnesky** [comment]
Hello everyone! Sorry for not responding earlier!

I've attempted to make most of the instrument presets have roughly the same perceived volume, although clearly I failed with "dubstep yoi yoi". Choosing an instrument preset does not override the instrument's volume slider, so you would have to manually reduce the volume after selecting this preset if you want to maintain the instrument's volume. (Fun fact: volume and panning are the only instrument settings that don't get overridden by selecting presets, so an instrument that is panned to the left will still be panned to the left after switching the preset!)

As part of my goal to maintain the perceived volume, the EQ and Note Filter controls also affect instrument volume. For example, if you add a low-pass filter, it cuts out some frequencies, which makes the instrument sound quieter overall, so BeepBox increases the overall volume of the instrument slightly in order to compensate for the low-pass. All of the filter types affect volume in various ways... but it's complicated and doesn't always work perfectly. The "dubstep yoi yoi" preset in particular has a pretty extreme Note Filter that increases the instrument volume at low frequencies, and the bitcrush effect extends that into the higher frequencies.

**Comment by The-Beesh-Spweesh** [comment]
I'm The Beesh-Spweesh. I obviously know to reduce the volume of the instrument.
